src/Features/Features.php

```diff
@@ -389,4 +400,17 @@ public static function add_admin_body_classes( $admin_body_class = '' ) {
 		$admin_body_class = implode( ' ', array_unique( $classes ) );
 		return " $admin_body_class ";
 	}
+
+	/**
+	 * Alias moved features classes to make them backward compatible.
+	 * We've moved our features class to src/Internal/Admin as part of The Merge.
+	 * See https://wp.me/p90Yrv-2HY for details.
+	 *
+	 * @param array $aliases an array of class aliases.
+	 */
+	private function register_moved_class_alisases( array $aliases ) {
+		foreach ( $aliases as $new_class => $orig_class ) {
```

Comment:
Why have the aliases as a class field? This only ever gets called with the same array, may as well co-locate the array within this function.